apps/settings/locales/settings.en-US.properties

```diff
@@ -1206,6 +1206,7 @@ hud-jsother=JS Other
 hud-dom=DOM
 hud-style=Style
 hud-other=Other
+hud-at=Advanced Telemetry
```

Comment:
Drive-by-nit: Please consider `hud-telemetry` for this l10n-id: Still relatively short, but the meaning becomes immediately obvious.